cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -0,0 +1,529 @@
+    protected static final String ENV_VAR_KAFKA_BRIDGE_SASL_USERNAME = "KAFKA_BRIDGE_SASL_USERNAME";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_SASL_MECHANISM = "KAFKA_BRIDGE_SASL_MECHANISM";
+
+    protected static final String ENV_VAR_KAFKA_BRIDGE_PRODUCER_CONFIG = "KAFKA_BRIDGE_PRODUCER_CONFIG";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_CONSUMER_CONFIG = "KAFKA_BRIDGE_CONSUMER_CONFIG";
+
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_ENABLED = "KAFKA_BRIDGE_AMQP_ENABLED";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_FLOW_CREDIT = "KAFKA_BRIDGE_AMQP_FLOW_CREDIT";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_MODE = "KAFKA_BRIDGE_AMQP_MODE";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_HOST = "KAFKA_BRIDGE_AMQP_HOST";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_PORT = "KAFKA_BRIDGE_AMQP_PORT";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_CERT_DIR = "KAFKA_BRIDGE_AMQP_CERT_DIR";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_AMQP_MESSAGE_CONNVERTER = "KAFKA_BRIDGE_AMQP_MESSAGE_CONNVERTER";
+
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_ENABLED = "KAFKA_BRIDGE_HTTP_ENABLED";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_HOST = "KAFKA_BRIDGE_HTTP_HOST";
+    protected static final String ENV_VAR_KAFKA_BRIDGE_HTTP_PORT = "KAFKA_BRIDGE_HTTP_PORT";
+
+    private KafkaBridgeTls tls;
+    private CertAndKeySecretSource tlsAuthCertAndKey;
+    private PasswordSecretSource passwordSecret;
+    private String username;
+    private String saslMechanism;
+    private KafkaBridgeHttpConfig http;
+    private boolean httpEnabled = false;
+    private boolean amqpEnabled = false;
+    private String bootstrapServers;
+    private KafkaBridgeConsumerSpec kafkaBridgeConsumer;
+    private KafkaBridgeProducerSpec kafkaBridgeProducer;
+
+    /**
+     * Constructor
+     *
+     * @param namespace Kubernetes/OpenShift namespace where Kafka Bridge cluster resources are going to be created
+     * @param cluster   overall cluster name
+     * @param labels    labels to add to the cluster
+     */
+    protected KafkaBridgeCluster(String namespace, String cluster, Labels labels) {
+        super(namespace, cluster, labels);
+        this.name = kafkaBridgeClusterName(cluster);
+        this.serviceName = name + "-service";
+        this.ancillaryConfigName = logAndMetricsConfigName(cluster);
+        this.replicas = DEFAULT_REPLICAS;
+        this.readinessPath = "/ready";
+        this.livenessPath = "/healthy";
+        this.livenessProbeOptions = DEFAULT_HEALTHCHECK_OPTIONS;
+        this.readinessProbeOptions = DEFAULT_HEALTHCHECK_OPTIONS;
+        this.isMetricsEnabled = DEFAULT_KAFKA_BRIDGE_METRICS_ENABLED;
+
+        this.mountPath = "/var/lib/bridge";
+        this.logAndMetricsConfigVolumeName = "kafka-metrics-and-logging";
+        this.logAndMetricsConfigMountPath = "/opt/bridge/custom-config/";
```

Comment:
I think this should be `/opt/strimzi` since that is there we install the bridge.

```suggestion
        this.logAndMetricsConfigMountPath = "/opt/strimzi/custom-config/";
```